Changelog

```diff
@@ -1,6 +1,49 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.4.0
-    - FEATURE: Add support for creating v2 torrents(requires libtorrent 2.0.x) (Chocobo1)
+    - FEATURE: Support for v2 torrents along with libtorrent 2.0.x support (glassez, Chocobo1)
+    - FEATURE: Support for Qt6 (glassez)
     - FEATURE: Expose libtorrent hashing_threads settings (Anton Bershanskiy)
+    - FEATURE: Add "Notification timeout" option (kevtechxx) 7722916fada4ae984d1c134edcd74edd5262ea4f
+    - FEATURE: Improve "Watched folders" feature (glassez) 2993fdb16954a9e3301efb7f9171ae1f03ab7743
+    - FEATURE: Add `connection_speed` to advanced settings (Chocobo1) ef79546508b2ac8c4ff2c5b7d62dc6215d60e53d
+    - FEATURE: Announce to all trackers if IP changed (#15001) (zhuangzi926) 2e8e2b04a141a8858726fc13714e2cb03956809
+    - FEATURE: Add tooltip for various columns (Chocobo1) bdc03b1c75074bc3c7950223c600d8139a7a9ef0
+    - FEATURE: Add context menu to toggle content tab columns (#15164) (AbeniMatteo) abd6eb2ff32504b48002ce69a89f10b21b30f461
+    - FEATURE: Add filter "Checking" to side panel (#15166) (AbeniMatteo) e5943b64c1cb803c52f69e1e837d73fd3f8b7281
+    - FEATURE: Add "Forced metadata downloading" state (#15185) (AbeniMatteo) 01d851440bc5f4b5dd913d94cea048ea184656dd
+    - FEATURE: Remember last viewed page in Options dialog (#15230) (Chocobo1) 2d4d2462680904ede2f9d46f080523683e6c1ef3
+    - FEATURE: Add tooltip to listening port spinbox (Chocobo1) 526ee9c9db99d49dbe7c4f84fc35438b92a48283
+    - FEATURE: Add "Skip hash check" option for watched folders (glassez) 90296b3ef09e479a00cd4b2f1969ecdcf8c0d5fe
+    - BUGFIX: Save "resume data" once file priority is changed (glassez) cd6959b712872407376b1cbd5bcc41daa4650d76
+    - BUGFIX: Show priority menu at top level if there is no other in Add New Torrent dialog (FozzeY) 3e47d26e440ac44ee511812fde253359b4351c17
+    - BUGFIX: Capitalize "peer flags" descriptions (Chocobo1) e38128119ca330a5a66aacbfebfcf1152d4f1d76
+    - BUGFIX: Reorder peer flags (Chocobo1) cb8d6a09395e2e3e81bff1bf9ff04dd3da9d81bb
+    - BUGFIX: Show "last activity" value under all circumstances (Chocobo1) 6864e13e6f0817567d9441028458f80d374c2afc
+    - BUGFIX: Correctly handle "no enough disk space" error (glassez) d4554c2e5c1fb0cdabd37e317a67b532c5987442
+    - BUGFIX: Elide text from the right for all columns' header (smigii) 8f02fe0cc606480c7d488bab8dcb4d85cbd1b08c
+    - BUGFIX: Fix startup with different profiles (jagannatharjun) a734199383b58f34420be2d6ee84562058f386ec
+    - WEBUI: Add reverse proxy source IP resolution (#15047) (HiFiPhile) f5315d9ba7692dded84cb3f0323f78ea8d50c103
```

Comment:
Aren't all these WebUI changes currently backported?..

Reply:
This one is not in 4.3.x. nor any other WebUI related, except for the ones i mention below.